docs/manuals/source/IntroductionAndTutorial/BareosWebui.rst

```diff
@@ -4,7 +4,282 @@
+
+     # Deny general command access
+     CommandACL = !.bvfs_clear_cache, !.exit, !configure, !purge, !prune, !reload
+     CommandACL = !create, !update, !delete, !disable, !enable
+     CommandACL = !show, !status
+
+     # Deny job related command access
+     CommandACL = !run, !rerun, !restore, !cancel
+
+     # Deny autochanger related command access
+     CommandACL = !mount, !umount, !unmount, !export, !import, !move, !release, !automount
+
+     # Deny media/volume related command access
+     CommandACL = !add, !label, !relabel, !truncate
+
+     # Deny SQL related command access
+     CommandACL = !sqlquery, !query, !.sql
+
+     # Deny debugging related command access
+     CommandACL = !setdebug, !trace
+
+     # Deny network related command access
+     CommandACL = !setbandwidth, !setip, !resolve
+
+     # Allow non-excluded command access
+     CommandACL = *all*
+
+     # Allow access to the following job resources
+     Job ACL = backup-bareos-fd, RestoreFiles
+
+     # Allow access to the following schedule resources
+     Schedule ACL = WeeklyCycle
+
+     # Allow access to the following catalog resources
+     Catalog ACL = MyCatalog
+
+     # Deny access to the following pool resources
+     Pool ACL = !Scratch
+
+     # Allow access to non-excluded pool resources
+     Pool ACL = *all*
+
+     # Allow access to the following storage resources
+     Storage ACL = File
+
+     # Allow access to the following client resources
+     Client ACL = bareos-fd
+
+     # Allow access to the following filset resources
+     FileSet ACL = SelfTest
+
+     # Allow access to restore to any filesystem location
+     Where ACL = *all*
+   }
+
+Alternatively the example above can be configured as following if you prefer a
+shorter version.
+
+.. code-block:: cfg
+   :caption: Profile Resource - Read-Only Access Example 2
+
+   Profile {
+     Name = "webui-user-readonly-example-2"
+
+     # Allow access to the following commands
+     CommandACL = .api, .help, use, version, status
+     CommandACL = list, llist
+     CommandACL = .clients, .jobs, .filesets, .pools, .storages, .defaults, .schedule
+     CommandACL = .bvfs_lsdirs, .bvfs_lsfiles, .bvfs_update, .bvfs_get_jobids, .bvfs_versions, .bvfs_restore
+
+     # Allow access to the following job resources
+     Job ACL = backup-bareos-fd, RestoreFiles
+
+     # Allow access to the following schedule resources
+     Schedule ACL = WeeklyCycle
+
+     # Allow access to the following catalog resources
+     Catalog ACL = MyCatalog
+
+     # Allow access to the following  pool resources
+     Pool ACL = Full, Differential, Incremental
+
+     # Allow access to the following storage resources
+     Storage ACL = File
+
+     # Allow access to the following client resources
+     Client ACL = bareos-fd
+
+     # Allow access to the following filset resources
+     FileSet ACL = SelfTest
+
+     # Allow access to restore to any filesystem location
+     Where ACL = *all*
+   }
+
```

Comment:
It might be an option to include the configuration files here by
.. literalinclude:: /include/config/backup-client1.cfg
   :language: bareosconfig

Reply:
for now I would like to leave it as it is